dev-libs/libportal/libportal-0.8.1.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+inherit flag-o-matic meson python-any-r1 vala virtualx
+
+DESCRIPTION="Flatpak portal library"
+HOMEPAGE="https://github.com/flatpak/libportal"
+SRC_URI="https://github.com/flatpak/libportal/releases/download/${PV}/${P}.tar.xz"
+
+LICENSE="LGPL-3"
+SLOT="0/1-1-1-1" # soname of libportal{,-gtk3,-gtk4,-qt5,-qt6}.so
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="gtk gtk-doc +introspection qt5 qt6 test +vala wayland X"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="
+	gtk-doc? ( introspection )
+	vala? ( introspection )
+"
+
+RDEPEND="
+	>=dev-libs/glib-2.58:2
+	introspection? ( dev-libs/gobject-introspection:= )
+	gtk? (
+		>=x11-libs/gtk+-3.24.41-r1:3[X?,wayland?]
+		>=gui-libs/gtk-4.12.5-r2:4[X?,wayland?]
+	)
+	qt5? (
+		dev-qt/qtcore:5=
+		dev-qt/qtgui:5=
+		dev-qt/qtx11extras:5=
+		dev-qt/qtwidgets:5=
+	)
+	qt6? (
+		dev-qt/qtbase:6=[X,gui,widgets]
+	)
+"
+DEPEND="${RDEPEND}
+	qt5? (
+		test? ( dev-qt/qttest:= )
+	)
+	qt6? (
+		test? ( dev-qt/qtbase:6 )
+	)
```

Comment:
- `:=` in DEPEND is a no-op, wrong anyway.
- the qt6 section redundant by RDEPEND anyway